Header header
int8 num_det
float64[] range
float64[] azimuth
float64[] elevation
float64[] rel_vel
float64[] rcs
float64[] range_var
float64[] az_var
float64[] el_var
float64[] vel_var
float64[] snr

